doc/tutorials/content/sources/qt_colorize_cloud/pclviewer.cpp

```diff
@@ -183,7 +206,11 @@ PCLViewer::lookUpTableChosen ()
 
   colorCloudDistances ();
   viewer_->updatePointCloud (cloud_, "cloud");
-  ui->qvtkWidget->update ();
+#if VTK_MAJOR_VERSION > 8
```

Comment:
You've extracted this kind of code into a separate function in other classes. Perhaps the same here as well?